src/features/usage-consolidation-accordion/summary-table/summary-table.js

```diff
@@ -88,38 +82,68 @@ const SummaryTable = ({
           <DropdownMenu
             role="menu"
           >
+            {entityType === entityTypes.PACKAGE ? (
+              <Button
+                id="summary-table-actions-view-titles"
+                buttonStyle="dropdownItem fullWidth"
+                role="menuitem"
+                onClick={handleViewTitles(onToggle)}
+                marginBottom0
+              >
+                <Icon
+                  icon="eye-open"
+                  size="small"
+                >
+                  <FormattedMessage id="ui-eholdings.usageConsolidation.summary.actions.view" />
+                </Icon>
+              </Button>
+            ) : null
+            }
             <Button
               buttonStyle="dropdownItem fullWidth"
               role="menuitem"
               onClick={onToggle}
               marginBottom0
             >
-              <FormattedMessage id="ui-eholdings.usageConsolidation.summary.actions.export" />
+              <Icon
+                icon="download"
+                size="small"
+              >
+                <FormattedMessage id="ui-eholdings.usageConsolidation.summary.actions.export" />
+              </Icon>
             </Button>
           </DropdownMenu>
         )}
       />
     )
   };
 
+  const customPropertiesWithFormatter = {
+    ...customProperties,
+    formatter: {
+      ...(customProperties.formatter || {}),
+      ...formatter,
+    },
+  };
+
   const contentData = rest.contentData || [{ cost, costPerUse, usage }];
 
   return (
     <KeyValue>
       <MultiColumnList
         id={id}
         contentData={contentData}
-        formatter={{
-          ...formatter,
-          ...customProperties.formatter,
-        }}
-        {...getSummaryTableColumnProperties(intl, customProperties)}
+        {...getSummaryTableColumnProperties(intl, customPropertiesWithFormatter, currency)}
```

Comment:
`currency` looks like the third leg here :)
Probably it looks like that we need to find better way in future.